tests/postgres/generator_test.go

```diff
@@ -338,6 +398,33 @@ var MpaaRating = &struct {
 }
 `
 
+var mpaaRatingEnumFileWithPkgName = `
+//
+// Code generated by go-jet DO NOT EDIT.
+//
+// WARNING: Changes to this file may cause incorrect behavior
+// and will be lost if the code is regenerated
+//
+
+package newenum
```

Comment:
There might be no need for the new test variables. We should be able to reuse existing one by just replacing `package enum` with `package newenum`.

Reply:
Okay, I'll replace the line `package enum` with the new package from existing variables then.